### PR TITLE
CLXP-174 Create new LP banner & replace CM Massive battle Quick access card on CM dashboard

### PR DESCRIPTION
**Pull request by HuyIT9898**
Purpose: Create new LP banner & replace CM Massive battle Quick access card on CM dashboard

review app: https://6628d2e0fd4aa63f3be2e1ab-gpnhtbdzfn.chromatic.com/?path=/story/template-backoffice-brandupdate--home

### CODE REVIEW

File: packages/@coorpacademy-components/src/molecule/go1-base-banner/test/fixtures/default.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+export default {
+  props: {
+    title: 'Learning Priorities, <br/> Focus on What Matters Most',
+    subTitle:
+      'Spotlight essential skills and direct learners to the most impactful training, <br/> optimizing their growth and enhancing your learning strategy.',
+    buttonLabel: 'Try Learning priorities',
+    onClick: () => console.log('click'),
+    image:
+      'https://s3-alpha-sig.figma.com/img/4c6f/87c0/99896b9146631494f8dea4f523e3cdff?Expires=1731888000&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=HvkcSCRSjz7RvLzB8D0HWNhuk7NUXFZSwLjBKmXxf5NPt-UCVf3wY9VUq~-bESu59PMJJVOcsdE-2h60AY65uswm-yk1yq2fd2dGvkeSXopK0YpqH-XLzdWo8FqHXVwSvB5RQlmA1KDHj1ERZjZJNijo2qXTE3eF71ksXxDm1g3c3XZgdD9TFfy8p9lLqKSBRpvway8bv7QNEcvuN-Q63-OR3EHHiftrZ-pOn0m2GFbuqIv3el-k3Zo60ZNINVeMjYkwN4~BG6PifpZdPWG9D3UenP12XNhPVCA3UJ7H1~ZTzu7q8jRG~h6TywD8S7sgowbtoQYoL5DE21bOxg0Ztg__'
```
Comment (AdrieanKhisbe) [inline packages/@coorpacademy-components/src/molecule/go1-base-banner/test/fixtures/default.js:9]: 🙈

File: packages/@coorpacademy-components/src/molecule/go1-base-banner/index.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ButtonLink from '../../atom/button-link';
+import style from './style.css';
+
+const Go1BaseBanner = props => {
```
Comment (AdrieanKhisbe) [inline packages/@coorpacademy-components/src/molecule/go1-base-banner/index.js:6]: Not sure about the name.
Would rather use Coorpmanager as a prefix: `CoorpmanagerBaseBanner`

File: packages/@coorpacademy-components/src/molecule/go1-base-banner/test/fixtures/default.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+export default {
```
Comment (AdrieanKhisbe) [inline packages/@coorpacademy-components/src/molecule/go1-base-banner/test/fixtures/default.js:1]: Can you put the existing banner as a fixture? `go1-catalog`

**Review by AdrieanKhisbe — CHANGES_REQUESTED** [review]

File: packages/@coorpacademy-components/src/molecule/go1-base-banner/test/fixtures/default.js (hunk 3 above)
Comment (HuyIT9898) [reply packages/@coorpacademy-components/src/molecule/go1-base-banner/test/fixtures/default.js]: The existing banner is a component name `go1-banner`. The styling was so different that I think I would better to create a new base banner

File: packages/@coorpacademy-components/src/molecule/go1-base-banner/test/fixtures/default.js (hunk 1 above)
Comment (HuyIT9898) [reply packages/@coorpacademy-components/src/molecule/go1-base-banner/test/fixtures/default.js]: Actually, the image I sent you to upload on s3 is unusable as it has a transparent background (i must have messed up the export 😆), and found out we can use correct image with this link. Can we keep it this way? Or we have to upload it to our s3 to use?

Comment (AdrieanKhisbe) [reply packages/@coorpacademy-components/src/molecule/go1-base-banner/test/fixtures/default.js]: we have to upload it to s3, s3-alpha-sig.figma.com we don't have control over it.
(also, please add the new image to the PR, and send it to me so I reupload it)

File: packages/@coorpacademy-components/src/molecule/go1-base-banner/test/fixtures/default.js (hunk 3 above)
Comment (AdrieanKhisbe) [reply packages/@coorpacademy-components/src/molecule/go1-base-banner/test/fixtures/default.js]: like cm-banner?
(that would be easier to customize if we want to highlight other features)

**Review by AdrieanKhisbe — APPROVED** [review]

**Pull request merged**